share/pedersen/dkg/dkg.go

```diff
@@ -248,7 +246,7 @@ func (d *DistKeyGenerator) ProcessJustification(j *Justification) error {
 // vss.Verifier.DealCertified()). If the distribution is certified, the protocol
 // can continue using d.SecretCommits().
 func (d *DistKeyGenerator) Certified() bool {
-	return len(d.QUAL()) >= d.t
+	return len(d.QUAL()) >= len(d.participants)
```

Comment:
This is needed currently to have every participants during the DKG time. I have a student working on the next iteration to be able to tolerate absent nodes. If you want to know the reason why, I'll be happy to explain but IRL ;)